### PR TITLE
feat: marketplace features removed from alpha build

### PR DESCRIPTION
**Pull request by ThisIsDemetrio**
## What this PR is for?

Removes from alpha build:

- `marketplace list-versions`
- `marketplace get` (only description updates)
- `marketplace delete` (only description updates)

From now on, these information (and the retrieval/deletion of marketplace items via itemId and tenantId) are available by default.

### CODE REVIEW

File: internal/cmd/marketplace/delete.go [hunk 1]
```diff
@@ -34,7 +33,7 @@ const (
 	// deleteItemByTupleEndpointTemplate formatting template for item deletion by the tuple itemID versionID endpoint; specify companyID, itemID, version
 	deleteItemByTupleEndpointTemplate = "/api/backend/marketplace/tenants/%s/resources/%s/versions/%s"
 
-	cmdDeleteAlphaLong = `Delete a single Marketplace item
+	cmdDeleteStableLong = `Delete a single Marketplace item
```
Comment (danielemarostica) [inline internal/cmd/marketplace/delete.go:36]: non metterei il riferimento a "stable"
```suggestion
	cmdDeleteLongDescription = `Delete a single Marketplace item
```

File: internal/cmd/marketplace/delete.go [hunk 2]
```diff
@@ -43,13 +42,7 @@ const (
...
 	Otherwise, it is preferable to pass the tuple companyId-itemId-version.
 	`
-	cmdDeleteStableLong = `Delete a single Marketplace item
-
-	You need to specify the ObjectID of the item with the flag object-id.
-	`
-
-	cmdDeleteAlphaUse  = "delete { --item-id item-id --version version } | --object-id object-id [flags]..."
-	cmdDeleteStableUse = "delete --object-id object-id [flags]..."
+	cmdDeleteStableUse = "delete { --item-id item-id --version version } | --object-id object-id [flags]..."
```
Comment (danielemarostica) [inline internal/cmd/marketplace/delete.go:45]: anche qui
```suggestion
	cmdDeleteUse = "delete { --item-id item-id --version version } | --object-id object-id [flags]..."
```

File: internal/cmd/marketplace/delete.go [hunk 3]
```diff
@@ -59,19 +52,10 @@ var (
...
-		long = cmdDeleteAlphaLong
-		use = cmdDeleteAlphaUse
-	} else {
-		long = cmdDeleteStableLong
-		use = cmdDeleteStableUse
-	}
-
 	cmd := &cobra.Command{
-		Use:        use,
+		Use:        cmdDeleteStableUse,
```
Comment (danielemarostica) [inline internal/cmd/marketplace/delete.go:56]:
```suggestion
		Use:        cmdDeleteUse,
```

File: internal/cmd/marketplace/delete.go [hunk 4]
```diff
@@ -59,19 +52,10 @@ var (
...
 		Short:      "Delete a Marketplace item",
-		Long:       long,
+		Long:       cmdDeleteStableLong,
```
Comment (danielemarostica) [inline internal/cmd/marketplace/delete.go:58]:
```suggestion
		Long:       cmdDeleteLongDescription,
```

File: internal/cmd/marketplace/get.go [hunk 5]
```diff
@@ -23,15 +23,14 @@ import (
...
-	"github.com/mia-platform/miactl/internal/util"
 	"github.com/spf13/cobra"
 )
 
 const (
 	getItemByObjectIDEndpointTemplate         = "/api/backend/marketplace/%s"
 	getItemByItemIDAndVersionEndpointTemplate = "/api/backend/marketplace/tenants/%s/resources/%s/versions/%s"
 
-	cmdGetAlphaLong = `Get a single Marketplace item
+	cmdGetStableLong = `Get a single Marketplace item
```
Comment (danielemarostica) [inline internal/cmd/marketplace/get.go:33]:
```suggestion
	cmdGetLongDescription = `Get a single Marketplace item
```

File: internal/cmd/marketplace/get.go [hunk 6]
```diff
@@ -40,29 +39,15 @@ const (
 	Passing the ObjectID is expected only when dealing with deprecated Marketplace items missing the itemId and/or version fields.
 	Otherwise, it is preferable to pass the tuple companyId-itemId-version.
 	`
-	cmdGetStableLong = `Get a single Marketplace item
-
-	You need to specify the ObjectID of the item with the flag object-id
-	`
-	cmdGetAlphaUse  = "get { --item-id item-id --version version } | --object-id object-id [FLAGS]..."
-	cmdGetStableUse = "get --object-id object-id [FLAGS]..."
+	cmdGetStableUse = "get { --item-id item-id --version version } | --object-id object-id [FLAGS]..."
```
Comment (danielemarostica) [inline internal/cmd/marketplace/get.go:42]:
```suggestion
	cmdGetUse = "get { --item-id item-id --version version } | --object-id object-id [FLAGS]..."
```

File: internal/cmd/marketplace/get.go [hunk 7]
```diff
@@ -40,29 +39,15 @@ const (
...
-		long = cmdGetAlphaLong
-		use = cmdGetAlphaUse
-	} else {
-		long = cmdGetStableLong
-		use = cmdGetStableUse
-	}
-
 	cmd := &cobra.Command{
-		Use:   use,
+		Use:   cmdGetStableUse,
```
Comment (danielemarostica) [inline internal/cmd/marketplace/get.go:48]:
```suggestion
		Use:   cmdGetUse,
```

File: internal/cmd/marketplace/get.go [hunk 8]
```diff
@@ -40,29 +39,15 @@ const (
...
 		Short: "Get Marketplace item",
-		Long:  long,
+		Long:  cmdGetStableLong,
```
Comment (danielemarostica) [inline internal/cmd/marketplace/get.go:50]:
```suggestion
		Long:  cmdGetLongDescription,
```

**Review by fredmaggiowski — APPROVED** [review]
LGTM

**Review by danielemarostica — APPROVED** [review]

**Review by JGiola — APPROVED** [review]

**Pull request merged**